plugins/woocommerce/includes/tracks/events/class-wc-settings-tracking.php

```diff
@@ -120,6 +141,12 @@ public function send_settings_change_event() {
 			}
 		}
 
+		if ( ! empty( $this->modified_options ) ) {
+			foreach ( $this->modified_options as $option_name => $selected_option ) {
+				$properties[ $option_name ] = $selected_option ?? '';
```

Comment:
I'm a little concerned that we aren't checking if the `$option_name` conflicts with any built-in/other Tracks event property names. But, we are already not doing this for `$toggled_options` `$state` above, and I suppose our options are all going to start with `woocommerce_` or `wc_`, so we are probably okay for now. Something to keep in mind in the future for tracking... we should ideally namespace/prefix these sort of things when we log them to avoid possible collisions.

Reply:
Also a little concerned that we don't do anything to make sure `$option_name` is a valid Tracks event property name (ensuring use of `_` and not `-`, etc.).

I'll open up a separate issue for us to make this a bit more robust, since this isn't an immediate concern.